Add TablestoreVectorStore.

xjtushilei opened the pull request:
Thank you for contributing to LangChain!

- [x] **PR title**:  community: add TablestoreVectorStore



- [x] **PR message**: 
    - **Description:** add TablestoreVectorStore
    - **Dependencies:** none


- [x] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration: yes
  2. an example notebook showing its use: yes

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

xjtushilei committed:
Add tablestore vector store

xjtushilei committed:
update example code

xjtushilei committed:
update score

xjtushilei committed:
Merge branch 'langchain-ai:master' into master

baskaryan committed:
Merge branch 'master' into master

baskaryan committed:
undo

baskaryan commented on docs/docs/integrations/providers/tablestore.mdx:
if this is provided by alibaba cloud should this info be consolidated in docs/docs/integrations/alibaba_cloud.mdx?

xjtushilei replied:
Thank you for your reminder. I have deleted the separate file and added it to `alibaba_cloud.mdx`.

baskaryan committed:
nit

baskaryan committed:
nit

baskaryan committed:
lint

xjtushilei committed:
consolidated in docs/docs/integrations/alibaba_cloud.mdx

xjtushilei committed:
Merge remote-tracking branch 'origin/master'

xjtushilei committed:
Merge branch 'master' into master

xjtushilei commented:
@baskaryan  Hello, I have already revised the previous comments. When you are free, can you help me take another look at the Review questions?

xjtushilei commented:
@baskaryan   Hello, I have already revised the previous comments. When you are free, can you help me take another look at the Review questions?

xjtushilei commented:
Hi， Any progress?

xjtushilei committed:
Merge branch 'master' into master

xjtushilei committed:
update code format

xjtushilei committed:
Merge remote-tracking branch 'origin/master'

xjtushilei committed:
update code format

xjtushilei committed:
fix type

xjtushilei committed:
update apu reference

xjtushilei committed:
fix test

xjtushilei committed:
fix docs

xjtushilei committed:
fix docs

xjtushilei commented:
This PR has not made any progress for a long time, so I updated it to the latest branch and modified some formatting issues according to CI. 

@baskaryan Please help merge this PR.
